releases/cosmwasm/2025-09-Solana-GMP-v1.0.0.md

```diff
@@ -0,0 +1,414 @@
+# Solana GMP Amplifier
+
+|                | **Owner**    |
+| -------------- | ------------ |
+| **Created By** | @nbayindirli |
+| **Deployment** | @nbayindirli |
+
+| **Axelar Env**       | **Deployment Status** | **Date**   |
+| -------------------- | --------------------- | ---------- |
+| **Devnet Amplifier** | -                     | 2025-09-18 |
+| **Stagenet**         | -                     | TBD        |
+| **Testnet**          | -                     | TBD        |
+| **Mainnet**          | -                     | TBD        |
+
+- [Amplifier Fork](https://github.com/eigerco/axelar-amplifier/tree/v0.1.0)
+- Contract Checksums:
+  - SolanaVotingVerifier: `52af1024c7548a724ec97728a1748bd1ce4ccb80bfb5c2a0ed7e57ee5ce5275c`
+  - SolanaGateway: `31572a174679ebbf31ac63fdfb99d7a99199d873b4558d61b8cfdf5800174fee`
+  - SolanaMultisigProver: `20c38aff9c725424cd71e7f85e357fb8a078954c11302f6d5e325060de9dac5c`
+
+## Background
+
+These are the instructions for deploying Amplifier contracts for Solana GMP connection.
+
+### Pre-requisites
+
+Ensure that the Solana gateway is deployed on Solana devnet/testnet/mainnet, as `VotingVerifier` needs the `sourceGatewayAddress` which is the External Gateway address.
+
+## Build and Store Contracts
+
+### Build Contracts
+
+1. Clone and checkout the correct branch:
+
+    ```bash
+    git clone --recurse-submodules https://github.com/eigerco/axelar-amplifier.git
```

Comment:
shouldn't we be deploying the v1 anchor version to include the changes to events and PDAs?